app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     package="com.example.lhi06.residue">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WRITE_INTERNAL_STORAGE"/>
```

Comment:
no i cant, i searched it online and a stack overflow had it in but not in the android developer pages